pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ConsumedLedgersTrimTest.java

```diff
@@ -142,6 +145,12 @@ public void testConsumedLedgersTrimNoSubscriptions() throws Exception {
         // the lastMessageId is still on the previous ledger
         restartBroker();
         // force load topic
+        Awaitility.await().untilAsserted(() -> {
+            try {
+                assertNotNull(pulsar.getBrokerService().getTopicIfExists(topicName).get(3, TimeUnit.SECONDS).get());
+            } catch (Exception ignored) {
```

Comment:
I think this should be removed since Awaitility catches RuntimeExceptions by default.

Reply:
Actually it does terminate the test, I just checked, so I'll take this back. The original approach was fine, my advice was bad. I'm sorry about that.

Reply:
Yes, an exception will still be thrown through try-catch assert, so I rolled back to the original way.

Reply:
`Awaitility.await().ignoreExceptions().untilAsserted` works. So one more round...

Reply:
@315157973  feel free to go with the original. this is my bad. Next time I'll know that it's `Awaitility.await().ignoreExceptions().untilAsserted` which should be used. :)

Reply:
I will use this way